### ISSUE TITLE
Per channel speed is broken

### ISSUE THREAD

**Issue by neon-dev** [andrewbrey/youtube-speed-memory#8]
https://github.com/andrewbrey/youtube-speed-memory/blob/231c322b692b39cabb571c1e68c281d7bbee454a/app/scripts.babel/youtube-speed-memory-listener.js#L75
The query doesn't find the channel, therefore `.innerText` throws a null pointer exception.

**Comment by andrewbrey** [comment]
@neon-dev Thanks for reporting this - fixing this up has been on my todo list (but quite low) for some time now and the main reason I've put fixing it off is because the extension is not listed in the store any more due to a takedown from Google ( see issue #6 ). I have a PR ( see issue #7 ) open that will address this issue, but I have not had time recently to complete this work. Thanks for using the extension, and bear with me as I find time to address the issue and get it re-submitted to the store! In the mean time, if you want to take a crack at the simple selector fix and open a PR, I would be more than happy to merge that in and possible start the re-submission process sooner.

Cheers!

**Comment by neon-dev** [comment]
Hi @andrewbrey. Thanks for your explanation and the fast response.
I installed it for Firefox, where it's still officially [available](https://addons.mozilla.org/en-US/firefox/addon/youtube-speed-memory/).
I actually specificially searched for an extension which can do this on a per channel setting, so I didn't really get to use it yet.
So I looked into #7 but ultimately didn't use the selectors there. The new code using channel ID instead of the name is a nice improvement, but i kept the fix for the old/current version fairly simple. See #9 
Didn't build and test the extension but the selectors work on video and channel pages, so everything should be fine.
